cpu/lm4f120/include/periph_cpu.h

```diff
@@ -115,6 +115,27 @@ typedef enum {
     ADC_RES_16BIT = 0xd00,            /**< not supported by hardware */
 } adc_res_t;
 
+
+/**
+ * @name SPI configuration
+ * @{
+ */
+#define SPI_NUMOF 4
```

Comment:
Yes, I was a little confuse by this style (looked anxiously for the endif until the end of the file)